### PR TITLE
Set typography 

### PR DESCRIPTION
**Pull request by gsambrotta**
Add Montserrat font, chose and add Esteban font for paragraph.
Set document, title and text style.
Adjust hero, logo and menu based on new font-base style.

Refering Issue: #23

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -130,4 +130,4 @@
 <!-- our external javascript scripts -->
 <script type="text/javascript" src="/assets/statics/polyfills/fetch.js"></script>
 <script type="text/javascript" src="/assets/statics/polyfills/template.js"></script>
-<script type="text/javascript" src="/assets/statics/polyfills/jsonp.js"></script>
\ No newline at end of file
+<script type="text/javascript" src="/assets/statics/polyfills/jsonp.js"></script>
```
Comment (Gushsd) [inline _includes/footer.html]: hmmm.. I don't really see the change in this line... Is it a deleted 'return'?

File: _includes/head.html [hunk 2]
```diff
@@ -4,7 +4,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
 
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}{% if page.tagline %} || {{page.tagline}}{%endif%} || OpenTechSchool.org</title>
-    <meta name="description" content="{{ site.description }}">
+    <meta name="description" content="{{ site.description }}" />
+	  <meta name="author" content="OpenTechSchool">
```
Comment (Gushsd) [inline _includes/head.html]: Is this correct that this line doesn't have the tag ending finalized with ' / '  while all the others do??

File: _includes/footer.html (hunk 1 above)
Comment (gsambrotta) [inline _includes/footer.html]: Is probably just the way Github shows how happy he is to have a new empty line at the end of the file

File: _includes/head.html (hunk 2 above)
Comment (gsambrotta) [inline _includes/head.html]: Yeah would be nicer if it has too.  `/` is semantically correct even if most of the modern browsers will fix it for us.
I would say is not a blocker but I will add it in the future for sure.

Comment (xMartin) [inline _includes/head.html]: With the document type we are using (`<!DOCTYPE html>`) the self-closing slash is not needed.

Comment (gsambrotta) [inline _includes/head.html]: 👍  true that

**Pull request merged**